docs/getting-started/setup/customizing-icons.md

```diff
@@ -127,11 +127,11 @@ You can read more about the `resolutions` field in the [pnpm documentation](http
 
 If you are using a bundler, you can use resolve aliases to replace the default `@ckeditor/ckeditor5-icons` package with your custom icon file. This method is similar to the one described above, but does not require creating a custom package.
 
-This section only shows selected bundlers as examples, but other bundlers support similar functionality out of the box or with first-party plugins.
+This section only shows selected bundlers as examples, but other bundlers support similar functionality out of the box or with third-party plugins.
```

Comment:
I meant first-party from bundler perspective, in a sense that there are official bundler plugins.